src/cms/models/events/event.py

```diff
@@ -15,19 +10,33 @@
 
 
 class Event(models.Model):
-    """Database object representing an event with name, date and the option to add recurrency.
+    """
+    Data model representing an event.
+
+    :param id: The database id of the event
+    :param start_date: The date when the event starts
+    :param start_time: The time when the event starts
+    :param end_date: The date when the event ends
+    :param end_time: The time when the event ends
+    :param picture: The thumbnail image of the event
+    :param archived: Whether or not the event is archived
+
+    Relationship fields:
+
+    :param region: The region to which the event belongs (related name: ``events``)
+    :param location: The point of interest where the event takes place (related name: ``events``)
+    :param recurrence_rule: If the event is recurring, the recurrence rule contains all necessary information on the
+                            frequency, interval etc. which is needed to calculate the single instances of an recurring
```

Comment:
Typo: "a recurring" instead of "an recurring" (yes, I'm pedantic from time to time :smile:)